JET_SELECTION_LOGIC.s

```diff
@@ -365,7 +365,7 @@ NOACY		CA	RWORD1		# Y-TRANSLATION NOT ACCEPTED
 # FAILURES.  IF THERE ARE BD FAILURES, Z-TRANSLATION COMMANDS WILL BE IGNORED, IN WHICH CASE THE ASTRONAUT SHOULD
 # SWITCH TO AC ROLL.
 #
-# NOTE THAT IF ONE QUAD FAILS (E.G. B FAILED), Z-TRANSLATION IS STILL POSSIBLE AND THAT THE UNDESIREABLE ROLL
+# NOTE THAT IF ONE QUAD FAILS (E.G. B FAILED), Z-TRANSLATION IS STILL POSSIBLE AND THAT THE UNDESIRABLE ROLL
```

Comment:
`UNDESIREABLE` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1047.jpg)